### PR TITLE
Remove Timed Control Tasks

### PR DESCRIPTION
**Pull request by ljgreenhill**
# Remove Timed Control Tasks

- Added retries if RockBLOCK serial sequence fails
- Remove clock manager and handle short cycles in main control loop
- Remove timed control task logic

### CODE REVIEW

**Review by ewz4 — APPROVED** [review]
Looks good!

File: src/ClockManager.cpp [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-    if (has_executed) {
-        sys_time_t earliest_start_time = TimedControlTaskBase::control_cycle_start_time + control_cycle_size;
-        cycle_dt = wait_until_time(earliest_start_time);
-    }
-    has_executed = true;
-    TimedControlTaskBase::control_cycle_start_time = get_system_time();
-    control_cycle_count++;
-    sfr::mission::cycle_no++;
-    sfr::mission::boot_time_mins = sfr::mission::cycle_no % (constants::time::one_minute / constants::timecontrol::control_cycle_time_ms);
-    sfr::mission::cycle_dur = constants::timecontrol::control_cycle_time - cycle_dt;
```
Comment (Duncan-McD) [inline src/ClockManager.cpp:23]: We should continue to track these three values in main control loop - personally I like having this stuff sill in clock manager, we could leave clock manager while still remove timed control tasks

**Review by Duncan-McD — CHANGES_REQUESTED** [review]
I think we should keep timing stuff out of main control loop in clock manager. We can still remove timed control tasks

Otherwise it all looks good

**Review by Duncan-McD — APPROVED** [review]
Looks good!

**Pull request merged**